internal/db/schema/migrations/postgres/9/02_oidc_managed_group.up.sql

```diff
@@ -0,0 +1,97 @@
+begin;
+
+create table auth_oidc_managed_group (
+  public_id wt_public_id
+    primary key,
+  auth_method_id wt_public_id
+    not null,
+  name wt_name,
+  description wt_description,
+  create_time wt_timestamp,
+  update_time wt_timestamp,
+  version wt_version,
+  filter wt_bexprfilter
+    not null,
+  -- Ensure that this managed group relates to an oidc auth method, as opposed
+  -- to other types
+  constraint auth_oidc_method_fkey
+    foreign key (auth_method_id) -- fk1
+      references auth_oidc_method (public_id)
+      on delete cascade
+      on update cascade,
+  -- Ensure it relates to an abstract managed group
+  constraint auth_managed_group_fkey
+    foreign key (public_id) -- fk2
+      references auth_managed_group (public_id)
+      on delete cascade
+      on update cascade,
+  constraint auth_oidc_managed_group_auth_method_id_name_uq
+    unique(auth_method_id, name),
+  -- QUESTION: is this necessary? It isn't for uniqueness, but maybe for an index to look up via auth method ID?
+  constraint auth_oidc_managed_group_auth_method_id_public_id_uq
+    unique(auth_method_id, public_id)
```

Comment:
This is not necessary here. It should be on the base table instead.

Reply:
Removed.

Reply:
Or rather, moved.